fineract-provider/src/main/java/org/apache/fineract/portfolio/loanproduct/serialization/LoanProductDataValidator.java

```diff
@@ -734,6 +737,16 @@ public void validateForCreate(final JsonCommand command) {
                     .value(receivablePenaltyAccountId).notNull().integerGreaterThanZero();
         }
 
+        if (!AccountingValidations.isAccrualBasedAccounting(accountingRuleType) && this.fromApiJsonHelper.parameterExists(LoanProductConstants.USE_BORROWER_CYCLE_PARAMETER_NAME, element)) {
```

Comment:
Wrong condition in the if (`LoanProductConstants.USE_BORROWER_CYCLE_PARAMETER_NAME`) ?